lib/appRelationsDiscovery.js

```diff
@@ -78,6 +79,8 @@ async function getRelations(asyncApiDocs, { syntax } = defaultOptions) {
     return metrics;
   if (syntax === 'mermaid')
     return getMermaidFlowchart(metrics);
+  if (syntax === 'plantUML')
```

Comment:
time for switch?

also what if I provide syntax `whatever`? some error?

would also be good to extend jsdoc with available options so I see them in IDE without checking out docs

Reply:
> time for switch?

Will change to switch in this PR and split functions for different diagrams in next one.

> also what if I provide syntax `whatever`? some error?

https://github.com/asyncapi/app-relations-discovery/blob/8798db4b63c26a629f1454708544635d1abfae9d/lib/appRelationsDiscovery.js#L20



> would also be good to extend jsdoc with available options so I see them in IDE without checking out docs

sure

Reply:
> would also be good to extend jsdoc with available options so I see them in IDE without checking out docs

Maybe we should switch to TS? :trollface: